server/tserver/src/main/java/org/apache/accumulo/tserver/TabletClientHandler.java

```diff
@@ -1009,6 +1009,9 @@ public void splitTablet(TInfo tinfo, TCredentials credentials, TKeyExtent tkeyEx
       } catch (IOException e) {
         log.warn("Failed to split " + keyExtent, e);
         throw new RuntimeException(e);
+      } catch (RuntimeException re) {
+        log.warn("Failed to split " + keyExtent, re);
```

Comment:
Was this code only added to log additional information?

Reply:
Yes, otherwise there is nothing in the log on the server side IIRC. The RuntimeException gets passed back to the client, but when that happens most of the detail is lost.